### DISCUSSION TITLE
About borrowing transactions

### DISCUSSION THREAD

**Discussion by FrankleeM** [firefly-iii/firefly-iii#6712] [Firefly III]
Borrowing transactions
According to the help documentation, two transactions are required

Borrowing (disbursement)
1. Asset account→ debt account (creditor's right)
2. Debt account → expenditure account (borrowing expenditure)

Collection (income)
1. Income account (collection) → debt account
2. Debt account → asset account

Existing problem: Two expenses and two incomes are recorded in the records
In the homepage board, the balance position will also be recorded twice, I want to ask if this is correct
![Snipaste_2022-12-11_15-51-21](https://user-images.githubusercontent.com/32917028/206893105-2f9b35a4-4aac-47e9-8792-20112892d908.png)

**Comment by JC5** [comment]
No, but this will be fixed in the new layout. Sorry for the confusing boxes.

**Reply by FrankleeM** [reply]
Thanks for the answer, and also thanks for providing this great product. It was really helpful for me to learn about financial management.
